learningequality/studio
# Welcome AI

## Summary

Adds a global getter that returns the value of AI feature flag.


### Manual verification steps performed
1. Import the isAIFeatureEnabled getter to a Vue component & set it as a computed property. 
2. Change status of ai feature flag for a user by admininstration user.
3. Render that component by logging in with that user and check computed properties.
4. isAIFeatureEnabled status should be reflected as it was set by admin. 
___
## Reviewer guidance

Taking a look at jest unit tests description to make sure the specs meets what we want.

## References
Closes https://github.com/learningequality/studio/issues/4276.

----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
